client/data/deposits/actions.js

```diff
@@ -1,9 +1,21 @@
 /** @format */
 
+/**
+ * External dependencies
+ */
+import { apiFetch, dispatch } from '@wordpress/data-controls';
+import { __, sprintf } from '@wordpress/i18n';
+import Currency from '@woocommerce/currency';
+import { addQueryArgs } from '@wordpress/url';
+
 /**
  * Internal Dependencies
  */
 import TYPES from './action-types';
+import { STORE_NAME } from '../constants';
+
+// TODO: Use the proper WCPay currency.
+const currency = new Currency();
```

Comment:
I think we should be using `import { formatCurrency } from 'utils/currency';` instead, until the proper WCPay currency will be used. What do you think?

Reply:
Good call, I agree. This has been updated.